scripts/install.sh

```diff
@@ -0,0 +1,226 @@
+print_question "Please enter the password for the system user (sudo user):"
+read -s user_password
+echo
+
+# Ask if user is going to use Wifi to connect to the network
+print_question "Will you use Wifi to connect to the network? (y/n):"
+wifi=$(get_yes_or_no)
+echo
+
+# If user is going to use Wifi, ask for the SSID and password
+if [[ $wifi == "y" ]]; then
+    print_question "Type the SSID of the network:"
+    read ssid
+    echo
+
+    print_question "Type the password of the network:"
+    read -s wifi_password
+    echo
+
+    print_question "Type the country code of the network (eg. DE, PT, ES, FR):"
+    read wifi_country_code
+    echo
+fi
+
+# As if user wants to introduce a custom timezone country code
+print_question "Do you want to introduce a custom timezone country code? (y/n):"
+custom_timezone=$(get_yes_or_no)
+echo
+
+# If user wants to introduce a custom timezone country code, ask for it
+if [[ $custom_timezone == "y" ]]; then
+    print_question "Please enter the timezone country code (eg. DE, PT, ES, FR):"
+    read timezone_country_code
+    echo
+
+    # Convert answer to uppercase
+    timezone_country_code=$(echo $timezone_country_code | tr '[:lower:]' '[:upper:]')
+else
+    timezone_country_code=$DEFAULT_TIMEZONE_COUNTRY_CODE
+fi
+
+# Ask if the user wants to override keyboard layout country code
+print_question "Override the keyboard layout country code? (y/n):"
+override_keyboard_layout=$(get_yes_or_no)
+echo
+
+# If user wants to override keyboard layout country code, ask for it
+if [[ $override_keyboard_layout == "y" ]]; then
+    print_question "Type the keyboard layout country code (eg. us, de, pt, es):"
+    read keyboard_layout_country_code
+    echo
+else
+    keyboard_layout_country_code=$DEFAULT_KEYBOARD_LAYOUT_COUNTRY_CODE
+fi
+
+# Ask if user wants to override default locale
+print_question "Override the default locale? (y/n):"
+override_locale=$(get_yes_or_no)
+echo
+
+# If user wants to override default locale, ask for it
+if [[ $override_locale == "y" ]]; then
+    print_question "Type the locale (eg. en_US, pt_PT, es_ES):"
+    read locale
+    echo
+else
+    locale=$DEFAULT_LOCALE
+fi
+
+# Ask if user wants to override the default hostname (edgebox)
+print_question "Override the default hostname (edgebox)? (y/n):"
+override_hostname=$(get_yes_or_no)
+echo
+
+# If user wants to override the default hostname, ask for it
+if [[ $override_hostname == "y" ]]; then
+    print_question "Type desired hostname:"
+    read hostname
+    echo
+
+else
+    hostname=$DEFAULT_HOSTNAME
+fi
+
+# Print a thank you message
+echo
+print_success "Thank you! The system will now be configured..."
+echo
+
+# Download the requested release from https://github.com/FooDeas/raspberrypi-ua-netinst/releases
+print_warning "Downloading raspberrypi-ua-netinst..."
+
+# Download the release zip file, deleting a previous one if it exists, and delete the extracted folder if it exists too
+rm -r ${RASPBERRYPI_UA_NETINST_RELEASE}.zip
```

Comment:
```suggestion
rm -rf ${RASPBERRYPI_UA_NETINST_RELEASE}.zip
```

I think we need f otherwise it'll complain

Reply:
Nice catch ;)